lib/parliament/decorators/party.rb

```diff
@@ -0,0 +1,10 @@
+
+module Parliament
+  module Decorators
+    module Party
+      def name
+        respond_to?(:partyName) ? partyName : []
```

Comment:
In the party.rb decorator, I think it would make more sense for the method name to return an empty string if Party does not respond to partyName.  This is because in the view you would be expecting a string and it might be confusing if you ended up with an empty array.